### PR TITLE
Add WebAuthn Example

### PR DESCRIPTION
**Pull request by kaden-stytch**
https://user-images.githubusercontent.com/95260470/145122672-7cd15fb3-25fc-427a-8516-d33a70d4ddf2.mov

https://user-images.githubusercontent.com/95260470/145122677-fcd3de46-2f77-4214-893b-fa4ffc126370.mov

### CODE REVIEW

**Comment by mary-stytch** [comment]
@kaden-stytch can you please update the screenshot in the README?

**Comment by mary-stytch** [comment]
> @kaden-stytch can you please update the screenshot in the README?

nvm, I see we'll need to update the image in our public CDN instead.

**Review by alex-stytch — APPROVED** [review]
LGTM – Approving from a WebAuthn API prospective

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ cp .env.template .env.local
 # Replace your keys in new .env.local file
 ```
 
-Next we'll configure the appropriate redirect URLs for your project, you'll set these magic link URLs for your project in the [Redirect URLs](https://stytch.com/dashboard/redirect-urls) section of your Dashboard. Add `http://localhost:3000/api/authenticate_magic_link` as both a login and sign-up redirect URL.
+Next we'll configure the appropriate redirect URLs for your project, you'll set these magic link URLs for your project in the [Redirect URLs](https://stytch.com/dashboard/redirect-urls) section of your Dashboard. Add `http://localhost:3000/api/authenticate_magic_link` and `http://localhost:3000/api/authenticate_magic_link_with_webauthn` as both a login and sign-up redirect URL.
```
Comment (chris-stytch) [inline README.md:40]: Not a blocker if you strongly prefer the wording here.

```suggestion
Next we'll configure the appropriate redirect URLs for your project, you'll set these magic link URLs for your project in the [Redirect URLs](https://stytch.com/dashboard/redirect-urls) section of your Dashboard. Add `http://localhost:3000/api/authenticate_magic_link` as both a login and sign-up redirect URL. If you'd like to try our [WebAuthn](https://stytch.com/docs/api/webauthn-overview) example integration, add `http://localhost:3000/api/authenticate_magic_link_with_webauthn` as a login and sign-up redirect URL as well.
```

I'm still mulling over how we want to do this long term as the complexity grows here, right now I'm leaning towards some sort of side bar instructions within the example app itself, but for now I think I want to target SDK EML and API SMS as the "default" integrations here, and the rest as "try these as well".

**Review by chris-stytch — CHANGES_REQUESTED** [review]
Works great locally and with `vercel dev`, but running into a timeout issue when sending the EML on Vercel, see https://stytch-nextjs-integration-291tr1ikm-stytch-auth.vercel.app/

The problem isn't obvious to me. @kaden-stytch let me know if you see something obvious that I missed. Going to move on to https://github.com/stytchauth/stytch-nextjs-integration/pull/47 so we can get that into this PR and circle back.

**Review by chris-stytch — APPROVED** [review]

**Pull request merged**